src/openstudio_app/OpenStudioApp.rc.in

```diff
@@ -1,6 +1,6 @@
 IDI_ICON1   ICON  DISCARDABLE  "${OPENSTUDIOAPPLICATION_ICON_FOLDER}/os.ico"
-IDI_ICON2   ICON  DISCARDABLE  "${OPENSTUDIOCORE_ICON_FOLDER}/osm.ico"
-IDI_ICON3   ICON  DISCARDABLE  "${OPENSTUDIOCORE_ICON_FOLDER}/osc.ico"
+IDI_ICON2   ICON  DISCARDABLE  "${OPENSTUDIOAPPLICATION_ICON_FOLDER}/osm.ico"
+IDI_ICON3   ICON  DISCARDABLE  "${OPENSTUDIOAPPLICATION_ICON_FOLDER}/osc.ico"
```

Comment:
Originally I'd done that on purpose FYI, thinking that:

* The OpenStudioApplication icon (os.ico) should probably come from here
* The filetypes (.OSM and .OSC) should probably come/be installed by the SDK itself.